query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/wfs/discover/DiscoverWfsSourcesTest.groovy

```diff
@@ -187,4 +184,25 @@ class DiscoverWfsSourcesTest extends Specification {
         } == 1
         report.messages()*.getPath() == [URL_FIELD_PATH]
     }
+
+    def prepareOpenSearchSourceUtils(int statusCode, String responseBody, boolean endpointIsReachable) {
+        def requestUtils = new TestRequestUtils(createMockFactory(statusCode, responseBody), endpointIsReachable)
+        def wfsUtils = new WfsSourceUtils()
+        wfsUtils.setRequestUtils(requestUtils)
+        return wfsUtils
+    }
+
+    def createMockFactory(int statusCode, String responseBody) {
```

Comment:
Unfortunately I had to copy paste this in each test class. Spock wouldn't let me create these mocks when I put it in SourceTestCommons.